README.md

```diff
@@ -114,6 +114,19 @@ Before running the tests make sure you are serving the app via `ng serve`.
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
 
+## Using convai-checker as a webcomponent
+
+Angular Elements is a new experimental API that allows exporting Angular
+components as webcomponents. This repo includes a demo of this functionality
+with the convai-checker component. To
+see it, do the following:
+1. Go into .angular-cli.json and replace "main.ts" with
```

Comment:
Inconsistent use of quotes. I suggest using code-quotes for filenames and code. i.e. .angular-cli.json becomes `.angular-cli.json`

Reply:
same for other names and URLs in here.